lib/ruby_lsp/test_reporting.rb

```diff
@@ -0,0 +1,80 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "json"
```

Comment:
Do we need to require `sorbet-runtime` to make sure `T::Sig` is available?

Reply:
Yes, added in `ruby_lsp_reporter.rb`.